pkg/ccl/backupccl/backup_test.go

```diff
@@ -2638,6 +2645,9 @@ func TestBackupRestoreDuringUserDefinedTypeChange(t *testing.T) {
 				typeChangesFinished := make(chan struct{})
 				_, sqlDB, _, cleanupFn := backupRestoreTestSetupWithParams(t, singleNode, 0, InitManualReplication, base.TestClusterArgs{
 					ServerArgs: base.TestServerArgs{
+
+						// Disabled under tenant bc AddInStatement-schema-only subtest panicked.
```

Comment:
s/bc/because/g

Can you mention the issue you've opened to track the investigation of this here?